sdk/src/test/scala/com/horizen/fork/ForkManagerUtil.scala

```diff
@@ -3,7 +3,7 @@ import scala.util.Try
 
 class ForkManagerUtil {
   def initializeForkManager(forkConfigurator:ForkConfigurator, networkName:String): Try[Unit] = Try {
-    ForkManager.reset()
+    ForkManager.networkName = null
```

Comment:
@MarcoOl94 check this line. Do we need it to be changed?

Reply:
No, reverted.